frontend/rolecall/src/app/performance/performance-editor.component.html

```diff
@@ -70,10 +70,7 @@
         </ng-container>
         <ng-template #submittedButtons>
           <div style="display: flex; flex-direction: row; justify-content: flex-end; width: 100%;">
-            <button mat-flat-button
-                class="button"
-                aria-label="New Performance"
-                (click)="onResetFromStart()">
+            <button mat-flat-button aria-label="New Performance" class="button" (click)="onResetFromStart($event)">
               new performance
             </button>
```

Comment:
See comment above about wide code. (If you believe the our deep code rule is wrong, let's have a discussion.)